ring-digest/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "ring-digest"
+version = "0.8.0"
+authors = ["RustCrypto Developers"]
+license = "MIT OR Apache-2.0"
+description = "RIPEMD-160 hash function"
+documentation = "https://docs.rs/ring-digest"
+repository = "https://github.com/RustCrypto/hashes"
+keywords = ["crypto", "ring", "hash", "digest", "sha1", "sha2", "SHA-1", "SHA-256", "SHA-384", "SHA-512", "SHA-512/256"]
```

Comment:
I think `SHA-*` keywords are redundant (none of the other crates use them), plus IIRC crates.io limits number of keywords to 5.